tests/Giraffe.Tests/Giraffe.Tests.fsproj

```diff
@@ -12,12 +12,17 @@
     <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="2.0.*" />
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="2.0.*" />
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.0.*" />
+
+    <PackageReference Include="Microsoft.AspNetCore.WebSockets" Version="2.0.*" />
```

Comment:
This is so dangerous. It should be pinned or converted to paket. Floating versions are such an anti pattern. Especially together with transitive deps.